examples/kyua.conf

```diff
@@ -43,6 +43,9 @@ syntax(2)
 -- Name of the system architecture (aka processor type).
 architecture = "x86_64"
 
+-- List of execution environments.
+execenv = "host jail"
+
```

Comment:
Yes, it will be skipped with `'jail' execenv is not supported or not allowed by the runtime user configuration` message. it follows the existing behavior of Kyua with the configuration variables like `architecture`.

And it covers a use case from Paul: https://lists.freebsd.org/archives/freebsd-hackers/2024-February/003020.html

Reply:
By the way, I was thinking about its name, that probably `execenvs` (plural) is better. Initially, I had thoughts that it could be hard to remember where `execenvS` and where `execenv` should be used. But the peers like `platform` or `architecture` are not plural and they do not accept a list of values, so, now, I think that it has more benefits to coin `execenvs` for the `kyua.conf` to remind about its list type like `execenvs = host docker qemu bhyve jail`. And the singular form is left as is on the test case side (`Kyuafile`).

So, it looks like it must be `execenvs` according to the existing design of Kyua. What do you think? It's kind of a decision which will be hard to change after GA point.